finish/src/main/java/io/openliberty/guides/bulkhead/global/eBank/microservices/Utils.java

```diff
@@ -103,7 +103,7 @@ public static String getHTMLForScheduleAppt() {
     }
 
     public static int calculateAdvisorNum(int requests) {
-        int localCounter = ((requests - 1) % global.eBank.microservices.BankService.bulkheadValue) + 1;
+        int localCounter = ((requests - 1) % io.openliberty.guides.bulkhead.global.eBank.microservices.BankService.bulkheadValue) + 1;
```

Comment:
Again BankService is in the same package, you don't need to include "io...microservices." part. You should be able to use BankService.bulkheadValue without the package.